l10n_ve_fiscal_book/invoice.py

```diff
@@ -182,11 +182,11 @@ def _get_doc(self, cr, uid, ids, name, args, context=None):
             ret.update({i: ''})
         if invs:
             for inv in invs:
-                if ((inv.type == "in_invoice" or inv.type == "out_invoice")
-                        and inv.parent_id):
+                if (inv.type == "in_invoice" or
+                        inv.type == "out_invoice") and inv.parent_id:
                     doc_type = "ND"
-                elif ((inv.type == "in_invoice" or inv.type == "in_refund")
-                      and inv.expedient):
+                elif (inv.type == "in_invoice" or
+                        inv.type == "in_refund") and inv.expedient:
                     doc_type = "E"
```

Comment:
if performing this changes,

Better do:
if inv.type in ('in_invoice', 'out_invoice') and inv.parent_id: